changelogs/fragments/3034-promox-kvm-return-new-id.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - proxmox_kvm - fix result of clone, should return newid instead of vmid
```

Comment:
```suggestion
  - proxmox_kvm - fix result of clone, now returns ``newid`` instead of ``vmid`` (https://github.com/ansible-collections/community.general/pull/3034).
```